### DISCUSSION TITLE
Meeting Edit page is on my staging site

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1529] [General]
Hi, I just moved my site from staging to live and when I try to edit a meeting now it is going to my staging site. Is that something I need to set or is that something I need to talk to my hosting company about?

Rose L
District 6 NC

**Comment by gkovats** [comment]
What's your site url? Are you trying to edit your meeting from the Admin Meetings list, or from the preview "Edit Meeting" link in the top admin bar when previewing it?

**Comment by district6wc** [comment]
This is our district site https://alanonalateen6nc.org/.  I am trying to edit from the link in the top bar, when I am signed in as admin. From the meeting list it seems to stay on site.

**Reply by gkovats** [reply]
I'll try recreating, but while you're looking, go to Wordpress settings and make sure the site url looks correct, that there isn't a staging url somewhere. If that looks good, then edit a meeting normally from the Admin meeting list, save, and then preview the page and try the top bar edit link there, and see if it goes to the correct location _after it's been saved once_. It could be an issue of your migrated posts hanging on to an old "guid" url, affecting your edit link.

**Comment by district6wc** [comment]
Hi

I did that and it all seems to be correctly pointing now. We do have a
staging site, I had just migrated it to the production server. Thanks for
the help.


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328
